docs/cloud-soar/incidents-triage.md

```diff
@@ -536,38 +443,22 @@ When creating incidents from Insights, adding additional required attributes to
 
 The Triage module is accessible from the Incidents section by clicking on Triage (or the name of the module if you have renamed it from the default of **Triage**). All events which have not been converted to an Incident will be displayed in a sortable table on the Triage main screen. Events may be sorted by any column values by clicking on the appropriate column.
 
-<img src={useBaseUrl('img/cloud-soar/image58.png')} alt="Events" width="800"/>
+<img src={useBaseUrl('img/cloud-soar/image58.png')} alt="Events" style={{border: '1px solid gray'}} width="800"/>
 
-The list of events can be filtered by any of the fields listed in the
-filter section at the top of the Triage main screen.
+The list of events can be filtered by any of the fields listed in the filter section at the top of the Triage main screen.
 
-<img src={useBaseUrl('img/cloud-soar/image181.png')} alt="Filter events" width="100"/>
+<img src={useBaseUrl('img/cloud-soar/image181.png')} alt="Filter events" style={{border: '1px solid gray'}} width="100"/>
 
 
-To view the details of a Triage event, click on the box and arrow icon
-in the Actions column for the event. If additional information is
-available, it will be displayed in this Event Details screen.
+To view the details of a Triage event, click on the box and arrow icon in the Actions column for the event. If additional information is available, it will be displayed in this Event Details screen.
 
-To begin triaging an event, click on the person icon in the Actions
-column for the event to **grab** the event. Once an event is grabbed by an
-analyst, any Playbooks defined for that incident type will be
-automatically executed and the results will be displayed in the Results
-section of the Event Details screen. Because all Playbooks for the
-specified incident type are automatically executed as soon as the
-incident is grabbed, it is recommended that separate incident types and Playbooks be created for Triage events.
+To begin triaging an event, click on the person icon in the Actions column for the event to **grab** the event. Once an event is grabbed by an analyst, any Playbooks defined for that incident type will be automatically executed and the results will be displayed in the Results section of the Event Details screen. Because all Playbooks for the
+specified incident type are automatically executed as soon as the incident is grabbed, it is recommended that separate incident types and Playbooks be created for Triage events.
```

Comment:
```suggestion
To begin triaging an event, click on the person icon in the Actions column for the event to **grab** the event. Once an event is grabbed by an analyst, any Playbooks defined for that incident type will be automatically executed and the results will be displayed in the Results section of the Event Details screen. Because all Playbooks for the specified incident type are automatically executed as soon as the incident is grabbed, it is recommended that separate incident types and Playbooks be created for Triage events.
```